### PR TITLE
Jlp-one-sentence-story

### PR DESCRIPTION
**Pull request by Jorge-Pioquinto**

### CODE REVIEW

File: in-a-world.md [hunk 1]
```diff
@@ -14,4 +14,5 @@ He falls terribly ill from eating mushrooms.
 
 Despite his illness, he was determined to uncover the truth behind the Locke family's secret, pushing himself to unravel the mystery even in the throes of his sickness.
 
-He propelled himself weakly back towards the vortex, clutching his stomach as it revolted violently against the mushrooms.
\ No newline at end of file
+After 10 years into his illness, he finally decided to go on a journey to find the best doctor and wizard in tbe world. That wizard's name was Hermione Granger.
```
Comment (thatcantbeso) [inline in-a-world.md:17]: the is spelled wrong

**Review by thatcantbeso — APPROVED** [review]
Great job

**Review by brendareyes-techprep — APPROVED** [review]
Wow, I heard she's good.

**Review by thierrychau — APPROVED** [review]
looks good

**Pull request merged**